xapian-applications/omega/worker.h

```diff
@@ -54,8 +56,8 @@ class Worker {
      *
      *  The assistant will not be started until it is necessary.
      */
-    Worker(const std::string& path)
-	: sockt(NULL), filter_module(path) { }
+    Worker(const std::string& path, bool verb)
+	: verbose(verb), sockt(NULL), filter_module(path) { }
```

Comment:
Don't just arbitrarily abbreviate a parameter name like this - it's especially confusing here since `verb` is also an English word, but means something different to `verbose`.

The Xapian convention for a parameter which initialises a member like this is just to append `_` to the parameter name (so `verbose_` here).